### PR TITLE
Add formatting for unfiltered gyroscope field

### PR DESCRIPTION
**Pull request by tbolin**
Adds formatting for the new black box field introduced in https://github.com/betaflight/betaflight/pull/12907

### CODE REVIEW

**Comment by nerdCopter** [comment]
screencap for reviewers.
using @tbolin's  BBL.
image

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by nerdCopter** [comment]
what is dev-team's consensus on verbiage of `unfiltered` versus `pre-filtered` (community usually speak "pre-filtered")?

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
